cluster/log-dump/log-dump.sh

```diff
@@ -63,8 +63,13 @@ readonly windows_node_otherfiles="C:\\Windows\\MEMORY.dmp"
 # file descriptors for large clusters.
 readonly max_dump_processes=25
 
+# Percentage of nodes that must be logexported successfully (otherwise the
+# process will exit with a non-zero exit code).
+readonly log_dump_expected_success_percentage="${LOG_DUMP_EXPECTED_SUCCESS_PERCENTAGE:-0}"
+
 # TODO: Get rid of all the sourcing of bash dependencies eventually.
 function setup() {
+  LOG_DUMP_EXIT_CODE=0
```

Comment:
I would lower case and move variable initialization outside of the function

Reply:
Done.